tests/scripts/all.sh

```diff
@@ -3878,6 +3878,19 @@ component_test_tls13_no_compatibility_mode () {
     tests/ssl-opt.sh
 }
 
+component_test_tls13_no_session_ticket() {
+    msg "build: default config with MBEDTLS_SSL_PROTO_TLS1_3, without session tickets"
+    scripts/config.py unset MBEDTLS_SSL_SESSION_TICKETS
+    scripts/config.py set MBEDTLS_SSL_PROTO_TLS1_3
```

Comment:
```suggestion
    scripts/config.py set MBEDTLS_SSL_PROTO_TLS1_3
    scripts/config.py set MBEDTLS_SSL_TLS1_3_COMPATIBILITY_MODE
```
otherwise most of the compatibility test cases are skipped. That's probably something we should improve but for the time being that's how it is.